### PR TITLE
fix(billing): remove pagination if empty list

### PR DESCRIPTION
**Pull request by Heiiji**
| Question         | Answer
| ---------------- | ---
| Branch?          |  `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| License          | BSD 3-Clause

## Description

just small fix when nothing is listed -> no pagination needed

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/billing/payment/refunds/billing-refunds.html [hunk 1]
```diff
@@ -216,6 +216,7 @@ <h2 data-translate="refunds_title2"></h2>
         </tbody>
     </table>
     <div
+        data-ng-show="refunds.list.results.length > 0"
```
Comment (radireddy) [inline packages/manager/apps/dedicated/client/app/billing/payment/refunds/billing-refunds.html:219]:
```suggestion
        data-ng-if="refunds.list.results.length > 0"
```

**Review by radireddy — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/billing/payment/refunds/billing-refunds.html (hunk 1 above)
Comment (frenautvh) [reply packages/manager/apps/dedicated/client/app/billing/payment/refunds/billing-refunds.html:219]: don't think ng-if will work here as pagination-server-side is still required to initialize and fetch data
ng-show seems appropriate

Comment (Heiiji) [reply packages/manager/apps/dedicated/client/app/billing/payment/refunds/billing-refunds.html:219]: it break the display with this method :/

Comment (Jisay) [reply packages/manager/apps/dedicated/client/app/billing/payment/refunds/billing-refunds.html:219]: yes the `div` must be in the dom in order to instanciate the directive on it. So `ngIf` won't work

**Review by Jisay — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**